server/graphql/v1/mutations/expenses.js

```diff
@@ -455,6 +471,12 @@ export async function payExpense(remoteUser, args) {
   if (expense.status === statuses.PAID) {
     throw new errors.Unauthorized('Expense has already been paid');
   }
+  if (expense.status === statuses.PROCESSING) {
+    throw new errors.Unauthorized('Expense has already been paid but is still being processed');
+  }
+  if (expense.status === statuses.ERROR) {
+    throw new errors.Unauthorized('Expense has already been paid and it failed, please create a new one');
```

Comment:
We should be able to retry the payment if it fails, but we can figure that out later in a separate issue

Reply:
Agreed, I think that that could be achieved by just removing this. I'll review and fix it now.